python/GangaDirac/Lib/Utilities/DiracUtilities.py

```diff
@@ -214,6 +214,34 @@ def _checkProxy( delay=60, renew = True, shouldRaise = True, force = False ):
             last_modified_time = time.time()
 
 
+def getAccessURLs(lfns):
```

Comment:
I would have said `DiracUtils` within the `Backend` would have been better, there's already functions like that there.
I would __really__ like to see the `DiracFile` making use of this to reduce the overhead of duplication of functionality.